### PR TITLE
simplify if else statement

### PR DESCRIPTION
**Pull request by foxyriver**
**What this PR does / why we need it**:

simplify if else statement

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by kargakis** [comment]
@k8s-bot bazel test this

File: pkg/kubectl/cmd/apply.go [hunk 1]
```diff
@@ -248,9 +248,9 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out, errOut io.Writer, opti
 				}
 				if uid, err := info.Mapping.UID(info.Object); err != nil {
 					return err
-				} else {
-					visitedUids.Insert(string(uid))
 				}
+				visitedUids.Insert(string(uid))
```
Comment (kargakis) [inline pkg/kubectl/cmd/apply.go]: uid is out of scope here

**Comment by fabianofranz** [comment]
A couple out of scope issues here, @foxyriver feel free to reopen when this is ready to review.

**Pull request closed without merging**